todo-app/src/todoState.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+class todoState extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: this.props.done
+    }
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    let changed = !this.state.checked
+    this.setState({
```

Comment:
Updating the state with `setState` is the way to go, nice 👍 You could have used the `checked` property of the event target, `event.target.checked`. So you don't depend on the previous value of the state. You might run in problems with this, as the state [updates asynchronously](https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous).